packages/@glimmer/runtime/test/attributes-test.ts

```diff
@@ -18,7 +18,7 @@ function compile(template: string) {
 
 function commonSetup() {
   env = new TestEnvironment(); // TODO: Support SimpleDOM
-  root = document.createElement('div');
+  root = document.getElementById('qunit-fixture')!;
```

Comment:
FWIW, I think this is not quite right.  There is an issue with qunits cleanup mechanism where attributes added to `qunit-fixture` are not properly cleared (they set inner HTML but do nothing to clear attributes).

I think what you want is:

```js
let testingRoot = document.getElementById('qunit-fixture');
root = testingRoot.createElement('div');
```

Reply:
For reference, this is done here: https://github.com/qunitjs/qunit/blob/bac08a415e514649d9ff30b82f5101c7fec739b0/runner/fixture.js#L37